src/lib/utilities/get-login-url.ts

```diff
@@ -0,0 +1,16 @@
+export const getLoginUrl = (
+  settings: Settings,
+  currentSearchParams: URLSearchParams,
+): string => {
+  const login = new URL('/auth/sso', settings.baseUrl);
+  const opts = settings.auth.options ?? [];
+
+  opts.forEach((option) => {
```

Comment:
after testing seems opts are not being added when making a request to `/auth/sso`

Reply:
fixed in the last commit